vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -41,7 +41,7 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"gopkg.in/gcfg.v1"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: only change in the file

Reply:
Interesting, it has been merged upstream this way. I removed it from this backport.